CITATION.cff

```diff
@@ -184,11 +184,11 @@ authors:
 
 
 cff-version: 1.2.0
-date-released: 2022-03-02
+date-released: 2022-03-10
```

Comment:
```suggestion
date-released: 2022-03-09
```

Reply:
Because of the failing tests I don't think I can do it today, that's why I put tomorrow in there :grimacing:

Reply:
sneaky. What failing tests? The GA ones?

Reply:
don't wait on those, we'll have to wait until they fix their package - as @zklaus points out - if it's too much bother we can temporarily turn the OSX tests off :beer:

Reply:
All right, fine for me :tada:

Reply:
Agreed. Though I hope everything will be resolved by tomorrow.

Reply:
what's that Bond movie... :grin: